mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -31,51 +31,16 @@ import org.apache.spark.sql.types.{ArrayType, StringType, StructType}
  */
 private[spark] object StopWords {
```

Comment:
This class is package private and hence users do not have access to `readStopWords`. We can either make it public or move the methods to `object StopWordsRemover`. If we take the former approach, we should check Java compatibility. I like the latter approach better.